charts/revere/templates/secretDefinition.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.secrets.gcpServiceAccount.secretsManager.enabled }}
+---
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
```

Comment:
Can you provide a link to this kind of manifest?

Reply:
Totally, this comes from [secrets-manager](https://github.com/tuenti/secrets-manager), we use it whenever we don't get secrets out of firecloud-develop and get it out of Vault directly. [Atlantis](https://github.com/broadinstitute/terra-helm/blob/master/charts/dsp-atlantis/templates/cleanup/secretdefinition.yaml), [duos](https://github.com/broadinstitute/terra-helm/blob/master/charts/duos/templates/secrets.yaml), [buffer](https://github.com/broadinstitute/terra-helm/blob/master/charts/buffer/templates/secrets.yaml), and [wsm](https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/templates/secrets.yaml) all use it, I'm sure there's other cases.